libr/core/cmd_anal.c

```diff
@@ -1162,11 +1162,18 @@ static int var_cmd(RCore *core, const char *str) {
 	switch (str[1]) { // afv[bsr]
 	case '\0':
 	case '*':
-	case 'j':
-		r_anal_var_list_show (core->anal, fcn, type, str[1]);
+	case 'j': {
```

Comment:
handle \0 and * differently, passing NULL in the last argument instead of creating the pj instance. i think this is the reason isi creating those newlines in travis

Reply:
Yeah, you are right. This removed a lot of newlines in the output. I made the changes.

Reply:
Yeah, you are right. This removed a lot of new lines from the output. I made the changes.